README.md

```diff
@@ -1,5 +1,60 @@
-Project: Movies Review Web App
+#Movies Review Web App
 
-Description: An app where users can see movies reviews and post their own reviews.
+An app where users can see movie reviews and post their own reviews.
 
-Authors: Curt Kim, Solomon Montagno, Jose Estrada# Group_5_Movies_Review_App
+##Authors
+
+Curt Kim, Solomon Montagno, Jose Estrada
+
+##Installation
+
+Clone this repository:
+
+```
+git clone https://github.com/dhk2431/Group_5_Movies_Review_App
+```
+
+Install the dependencies:
+
+```
+cd Group_5_Movies_Review_App
+yarn install
+```
+
+##Usage
+
+To run the server:
+
+```
+yarn dev
+```
+
+To make the database:
+
+```
+createdb movies-review-app_development
+createdb movies-review-app_test
+createdb movies-review-app_e2e
+
+yarn migrate:make movies
+yarn migrate:latest
+```
+
+##[Text for the link here](URL here!)
+
+##Features:
+
+###List of movies
+
+###A Nav-bar
+
+###User authentication
+
+###Clicking the movies lets you see its details and submit a review if you are a registered user
+
+###You can leave a comment under a review
+
+##Technologies in the project here:
+
+###Technology1
+###Technology2
```

Comment:
these headers should have a space between the # and the text
Most of these I would actually turn into bullets though
no dummy text (i.e. technology 1, etc)